song-spotter-50f1d-0801ae9be295.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can you delete this file? I made a new key so no worries about by accidentally pushing this